corebehrt/azure/log.py

```
@@ -0,0 +1,141 @@
+from contextlib import contextmanager
+
+MLFLOW_AVAILABLE = False
+
+try:
+    # Try to import mlflow and set availability flag
+    import mlflow
+
+    MLFLOW_AVAILABLE = True
+except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)